assets/js/app.js

```diff
@@ -184,7 +225,8 @@ function App({ uniqueId, socket }) {
     function getToken() {
         const now = Math.floor(Date.now() / 1000);
         const expire = parseInt(localStorage.getItem('expire'));
-        if (expire === null || parseInt(expire) <= now) {
+
+        if (expire === null || expire <= now) {
```

Comment:
This should also take expire into (on hour before into account)? So if expire is less that one hour it's expired. Maybe the one hour timeout should be configured in one place!

Reply:
If it is not expired it is still valid and can be used. The refresh of the token is handled by setupTokenRefresh()